testing/travis-fork-check.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
+import os
+import subprocess
+import sys
+
+def check_travis_repo_slug():
+    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
+        logging.warning("Decryption disabled. Setting MANTL_CI_FORK_CHECK to 1")
+        print("1")
+    else:
+        logging.info("Decrypting OS ssh key")
```

Comment:
So… why is decrypting a SSH key in the script to detect if the build is a fork? They ought to be separate.